### PR TITLE
libusb: version bumped to 1.0.24.

### PR DESCRIPTION
**Pull request by florin65**

### CODE REVIEW

File: libs/libusb/DETAILS [hunk 1]
```diff
@@ -1,12 +1,12 @@
           MODULE=libusb
-         VERSION=1.0.23
+         VERSION=1.0.24
           SOURCE=$MODULE-$VERSION.tar.bz2
 SOURCE_DIRECTORY=$BUILD_DIRECTORY/$MODULE-$VERSION
       SOURCE_URL=http://github.com/libusb/libusb/releases/download/v$VERSION/
```
Comment (stumbles) [inline libs/libusb/DETAILS:5]: Perhaps changing the source url to;

SOURCE_URL_FULL=https://github.com/libusb/libusb/archive/v$VERSION.tar.gz

would fix the problem

**Comment by florin65** [comment]
Retest this please

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Pull request merged**